modules/integration/tests-common/admin-clients/pom.xml

```diff
@@ -54,5 +54,20 @@
             <artifactId>org.wso2.carbon.bpmn.stub</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.2.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpmime</artifactId>
+            <version>4.3.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20080701</version>
+        </dependency>
     </dependencies>
 </project>
```

Comment:
Fixed in the latest commit, moved the dependencies to parent pom and set scope as test.